code/wdb/writedown.q

```diff
@@ -0,0 +1,14 @@
+/-endofperiod function to savedown by integer
+endofperiod:{[currp;nextp;data]
+	.lg.o[`endofperiod;"Received endofperiod. currentperiod, nextperiod and data are ",(string currp),", ", (string nextp),", ", .Q.s1 data]
+        / Need to download sym file to scratch directory if this is Finspace application
```

Comment:
You can assume this is a finspace application as this code is only loaded in finspace (so redundant comment)